products/compute/ansible.yaml

```diff
@@ -141,11 +141,11 @@ overrides: !ruby/object:Provider::ResourceOverrides
   TargetPool: !ruby/object:Provider::Ansible::ResourceOverride
     provider_helpers:
       - 'products/compute/helpers/provider_target_pool.py'
+  TargetVpnGateway: !ruby/object:Provider::Ansible::ResourceOverride
+    exclude: false
```

Comment:
Should the resourceoverride just go away in this case?

Reply:
https://github.com/GoogleCloudPlatform/magic-modules/blob/8e47e682ae8c6d0c1cd1641f8e3a1aabbb7a65d1/products/compute/api.yaml#L3150

[Right now](https://github.com/GoogleCloudPlatform/magic-modules/blob/8e47e682ae8c6d0c1cd1641f8e3a1aabbb7a65d1/products/compute/api.yaml#L3150), TargetVpn is exclude: true by default.

I'm not a huge fan of placing exclude: true in api.yaml in this case, as opposed to placing exclude: true in puppet/chef/ansible. Maybe bring this up at some point?

Reply:
Oh, you're right. Yeah I've definitely done it- it makes it easy in development to get something out the door for just a single provider, but it should probably be treated a bit more like a hack.